src/test/java/org/folio/utility/VertxUtility.java

```diff
@@ -0,0 +1,171 @@
+package org.folio.utility;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.support.HttpClient;
+import org.folio.rest.support.Response;
+import org.folio.rest.support.ResponseHandler;
+import org.folio.rest.tools.utils.NetworkUtils;
+
+public class VertxUtility {
+  private static final Logger logger = LogManager.getLogger();
+  private static Vertx vertx;
+  private static HttpClient client;
+  private static int port = 0;
+
+  private VertxUtility() {
+    throw new UnsupportedOperationException("Cannot instantiate utility class.");
+  }
+
+  public static HttpClient getClient() {
+    return client;
+  }
+
+  public static void startVertx(String tenantId)
```

Comment:
This doesn't really only start vertx, it also deploys the RMB verticle and enable the module for the tenant.

I think it could be easier to follow if the name of this describes these responsibilities more explicitly.